packages/dev/core/src/Sprites/sprite.ts

```diff
@@ -213,10 +213,12 @@ export class Sprite extends ThinSprite implements IAnimatable {
         sprite.isVisible = parsedSprite.isVisible;
         sprite.useAlphaForPicking = parsedSprite.useAlphaForPicking;
 
-        sprite.fromIndex = parsedSprite.fromIndex;
-        sprite.toIndex = parsedSprite.toIndex;
-        sprite.loopAnimation = parsedSprite.loopAnimation;
-        sprite.delay = parsedSprite.delay;
+        sprite._fromIndex = parsedSprite.fromIndex;
+        sprite._toIndex = parsedSprite.toIndex;
+        sprite._loopAnimation = parsedSprite.loopAnimation;
+        sprite._delay = parsedSprite.delay;
+
+        sprite.playAnimation(sprite._fromIndex, sprite._toIndex, sprite.loopAnimation, sprite.delay);

```

Comment:
this will now be called twice, you should use the latest fix proposal from @Popov72